Source/SA/DPD/DPD.csproj

```
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{C725B272-463E-421D-8648-4E3F87868711}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>DPD</RootNamespace>
+    <AssemblyName>DPD</AssemblyName>
```

This can be configured via the property project or by editing the raw XML. Either way, it must be 
```suggestion
    <AssemblyName>NationalInstruments.ReferenceDesignLibraries.SA.DPD</AssemblyName>
```

---

@mdwhitten DPD uses both SA and SG. are we sure it should live here?

---

Hmm. Good question. Because RFmx DPD is of course a property of RFmx which thus far lives under SA I was fine with leaving it there. But you're right that it really involves both.

Power Servo and ET are going to live in a separate folder path and namespace; I think we should come up with a consensus on nomenclature but these are unified by the idea of being "Test Methods". Hence for power servo I placed it in the /Source/Methods/ directory and used the namespace _NationalInstruments.ReferenceDesignLibraries.Methods_. I think we should do the same here.

---

@mdwhitten, please review 8ac5b20 and resolve this conversation if fixed.

---

Thanks @cindyxie! The namespace is now correct, but could you also move the code itself to the following directory: _\Source\Methods\RFmxDPD_?

---

Please review this commit 67fcb5e